src/www/ui/api/documentation/openapi.yaml

```
@@ -1544,6 +1544,61 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
           $ref: '#/components/responses/defaultResponse'
+  /file/upload{id}/item/{itemId}/copyright/{hash}:
```

Be consistent with other endpoints
```suggestion
  /uploads/{id}/item/{itemId}/copyrights/{hash}:
```